### DISCUSSION TITLE
Remove Content Based Package restrictions.

### DISCUSSION THREAD

**Discussion by maxy1** [lxhao61/integrated-examples#203] [General]
Hi, Can you please tell me how to remove Content Based Package restrictions using xray ?
For example we have zoom packages with unlimited data. People use zoom.us as the SNI to bypass the traffic with tls and allowinsecure checked.  I think they are using Xray to achieve this. I have been trying to find good config for past two weeks none is successful. ( its mainly due to lack of my knowledge and understanding. ).
if you can give me bit of clues example configs I really appreciate.

**Comment by lxhao61** [comment]
不完全明白你在说什么？据我猜测本人示例中``Xray 的 REALITY H2/gRPC 应用``或``Xray 的 REALITY Vision 应用``配置应该可满足你需求。
